v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/datastream/ChangeEventTypeConvertorTest.java

```diff
@@ -203,6 +203,81 @@ public void cannotConvertNonExistentRequiredFieldToLong() throws Exception {
         ChangeEventTypeConvertor.toLong(ce, "field1", /* requiredField= */ true), new Long(123457));
   }
 
+  /*
+   * Tests for float conversion.
+   */
+  @Test
+  public void canConvertToFloat() throws Exception {
+
+    // Change Event with all valid forms of double values
+    JSONObject changeEvent = new JSONObject();
+    changeEvent.put("field1", 123456789f);
```

Comment:
Add a field with highest float number and one with highest precision supported in spanner

Reply:
Added float min, max, +inf and -inf test cases.